csharp/GrpcOpenTracing/GrpcOpenTracing/GrpcTraceLogger.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using OpenTracing;
+
+namespace GrpcOpenTracing
+{
+    internal class GrpcTraceLogger<TRequest, TResponse> 
+        where TRequest : class 
+        where TResponse : class
+    {
+        private readonly ISpan span;
+
+        public GrpcTraceLogger(ISpan span)
+        {
+            this.span = span;
+        }
+
+        public void Request(TRequest req)
+        {
+            this.span.Log(new Dictionary<string, object>
+            {
+                { LogFields.Event, "gRPC request" },
+                { "data", req }
```

Comment:
Logging the whole request seems wrong. Would be best to compare this with the Java implementation and check what fields they log.

Reply:
Java is also logging the message, but only if it's streaming or if verbose flag is set: [ServerTracingInterceptor.java](https://github.com/grpc-ecosystem/grpc-opentracing/blob/master/java/src/main/java/io/opentracing/contrib/ServerTracingInterceptor.java#L121). I also want to add a verbose flag later on.